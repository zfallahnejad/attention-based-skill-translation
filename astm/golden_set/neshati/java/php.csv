157882,php
22656,php
18393,php
