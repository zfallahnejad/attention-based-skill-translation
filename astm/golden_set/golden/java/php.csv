157882,19,14,0.5757575757575758
22656,13,7,0.65
18393,11,8,0.5789473684210527
